### PR TITLE
done with edit component

### PR DESCRIPTION
**Pull request by DoctorFreed**

### CODE REVIEW

File: src/app/edit-window/edit-window.component.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    </mat-form-field>
+  </div>
+</div>
+<div mat-dialog-actions>
+  <button mat-button (click)="onNoClick()">Отмена</button>
+  <button
+    mat-button
+    cdkFocusInitial
+    [disabled]="editForm.invalid"
+    [mat-dialog-close]="{
```
Comment (usatovaDasha) [inline src/app/edit-window/edit-window.component.html:53]: maybe use function?

File: src/app/reception-table/reception-table.component.ts [hunk 2]
```diff
@@ -9,10 +11,35 @@ import { ReceptionService } from '../reception.service';
...
+  }
+
+  openDialog(element: any): void {
+    const dialogRef = this.dialog.open(EditWindowComponent, {
+      width: '300px',
+      data: { ...element },
+    });
+
+    dialogRef.afterClosed().subscribe((result) => {
+      console.log(result);
```
Comment (usatovaDasha) [inline src/app/reception-table/reception-table.component.ts:30]: remove console

File: src/app/reception-table/reception-table.component.ts [hunk 3]
```diff
@@ -9,10 +11,35 @@ import { ReceptionService } from '../reception.service';
...
+      if (result) {
+        this.receptionService
+          .updateReception(
```
Comment (usatovaDasha) [inline src/app/reception-table/reception-table.component.ts:33]: why don't send only result?

Comment (DoctorFreed) [reply src/app/reception-table/reception-table.component.ts:33]: not working :( typescript...

File: src/app/reception-table/reception-table.component.ts [hunk 4]
```diff
@@ -3,6 +3,14 @@ import { ReceptionService } from '../reception.service';
 import { MatDialog } from '@angular/material/dialog';
 import { EditWindowComponent } from '../edit-window/edit-window.component';
 
+interface IReception {
```
Comment (usatovaDasha) [inline src/app/reception-table/reception-table.component.ts:6]: make this interface global and import it

**Pull request merged**